ilastik/applets/tracking/base/trackingBaseDataExportGui.py

```diff
@@ -188,33 +188,6 @@ def set_default_export_filename(self, filename):
         # TODO: remove once tracking is hytra-only
         self._default_export_filename = filename
 
-    # override this ExportingOperator function so that we can pass the default filename
-    def show_export_dialog(self):
-        """
-        Shows the ExportObjectInfoDialog and calls the operators export_object_data method
-        """
-        # TODO: remove once tracking is hytra-only
```

Comment:
Thanks a lot for attempting this fix! There's just one problem: This code must still work with pgmlink so that windows users still get a working tracking workflow. This is why the **TODOs** are in there and the functions still exist.

So please do not remove them yet. (You can try that by installing the `pgmlink` package in your environment and temporarily moving the `hytra` folder from your conda environment's `site-packages` folder.)